### DISCUSSION TITLE
How to compile for windows?

### DISCUSSION THREAD

**Discussion by fredericgoossens** [ivangabriele/clamav-desktop#63] [Questions & Answers]
I have installed make & rust

Tried to compile using the cargo.toml file in /src-tauri/  ('cargo build' command in windows), but it doesn't seem to create a release folder.
there is a clamav-desktop.exe binary. however, when opening this, it just opens some Chrome dev tools and Edge.

Sorry, I'm new to Rust and confused about how to compile it. Do I have to run each cargo.toml file separately?

**Comment by ivangabriele** [comment]
Hi!

Sorry for the late answer but did you follow the instructions [there](https://github.com/ivangabriele/clamav-desktop/blob/main/CONTRIBUTING.md#windows-8-msi)?

(I convert that issue to a discussion since it's a question rather than an issue)
